doc/developers/advanced_installation.rst

```diff
@@ -188,8 +188,7 @@ to enable OpenMP support:
 
 - or install `libomp` with Homebrew to extend the default Apple clang compiler.
 
-For Apple Silicon M1 hardware, only the conda-forge method below is known to
-work at the time of writing (January 2021). You can install the `macos/arm64`
+You can install the `macos/arm64`
```

Comment:
I would remove this full sentence. This was related to the for Apple M1, only the conda approach works, which is no longer true.

Reply:
Done.